tests/DefinitionTest.elm

```diff
@@ -0,0 +1,44 @@
+module DefinitionTest exposing (suite)
+
+import Expect exposing (Expectation)
+import Html
+import Html.Attributes
+import Html.Styled
+import Page.Definition exposing (view)
+import Test exposing (Test, describe, test)
+import Test.Html.Event as Event
+import Test.Html.Query as Query
+import Test.Html.Selector exposing (attribute, tag, text)
+
+
+suite : Test
+suite =
+    describe "Definition View"
+        [ test "Definition view has title" <|
+            \() ->
+                view {}
+                    |> Html.Styled.toUnstyled
+                    |> Query.fromHtml
+                    |> Query.contains [ Html.text "Definition page" ]
```

Comment:
Maybe just that it has an h1?

Reply:
I am only demo-ing when we so the real content we could use test ID strategy - or copy doc and also will be more interesting when there is a model and updates